packages/yarnpkg-core/sources/Installer.ts

```diff
@@ -21,7 +21,30 @@ export type FinalizeInstallStatus = {
   buildDirective: Array<BuildDirective>,
 };
 
+export type PackageMeta = Record<any, any>;
```

Comment:
I'd prefer to make it clear that this structure is tied to installers rather than being a set of general-purpose metadata.

```suggestion
export type InstallerPackageMeta = Record<unknown, unknown>;
```